groups/infrastructure/variables.tf

```diff
@@ -113,3 +119,9 @@ variable "create_route53_aliases" {
   description = "Whether to create Route53 aliases pointing to the ALB"
   type        = bool
 }
+
+variable "route53_aliases_ch_service_mock" {
+  type        = list(string)
+  description = "The Route53 aliases to create for ch-service-mock lb ."
```

Comment:
Other descriptions don't include a full stop, can be removed. Also there's a space between the full stop and last word